batch_finder_resource_method.md

```diff
@@ -0,0 +1,418 @@
+---
+layout: api_reference
+title: Batch Finder Resource Method
+permalink: /batch_finder_resource_method
+excerpt: This documentation describes how restli framework support batch of search requirement.
+---
+
+# Batch Finder Resource Method
+
+## Contents
+
+-   [Overview](#overview)
+-   [Protocol](#protocol)
+    -   [Request](#request)
+        -   [URI](#uri)
+        -   [Java Request Builders](#java-request-builders)
+    -   [Response](#response)
+        -   [Reponse Body](#batchcollectionresponse)
+        -   [Error Handling](#error-handling)
+-   [Resource API](#resource-api)
+    -   [Method Annotation and Parameters](#method-annotation-and-parameters)
+    -   [Criteria Filter](#criteria-filter)
+    -   [BatchFinderResult](#batchfinderresult)
+-   [Future consideration](#future-consideration) 
+    
+## Overview   
+The BATCH_FINDER allows combining multiple requests to one FINDER within a Rest.li resource into a single call. 
```

Comment:
I know I am the author of this sentence :) but by reading it again, I feel like the reader might understand that batch-finder will allow batching finder methods, like a multiplexer. In fact, we support a new resource method that accepts a list of filters set.  Instead of callings multiple finders with different filter values, we can call 1 batch-finder method with a list of filters.